### PR TITLE
[WIP]Base template on twbs

### PR DESCRIPTION
**Pull request by ahonymous**

### CODE REVIEW

**Comment by spolischook** [comment]
includes/content.html.twig - is not a good name

**Comment by ahonymous** [comment]
ok, renemed to section is good?

File: app/config/assetic.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    bundles:
+        - GeekhubResourceBundle
+    filters:
+        cssrewrite: ~
+        yui_css:
+            jar: %kernel.root_dir%/../vendor/nervo/yuicompressor/yuicompressor.jar
+    assets:
+        jquery:
+            inputs:
+                - %kernel.root_dir%/../vendor/frameworks/jquery/jquery.min.js
```
Comment (spolischook) [inline app/config/assetic.yml]: Merge all js files in one scripts.js file

File: src/Geekhub/ResourceBundle/Resources/views/layout.html.twig [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+        <![endif]-->
+    </head>
+
+    <body>
+        {% block body %}
+        {% endblock %}
+
+        <!-- Bootstrap core JavaScript
```
Comment (spolischook) [inline src/Geekhub/ResourceBundle/Resources/views/layout.html.twig]: Remove comments that not provided to your themplate

File: src/Geekhub/ResourceBundle/Resources/views/layout.html.twig [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="">
```
Comment (spolischook) [inline src/Geekhub/ResourceBundle/Resources/views/layout.html.twig]: Add block description with default value, like "CheDream - Open source crowdfunding system"

File: src/Geekhub/ResourceBundle/Resources/views/layout.html.twig [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        <meta name="author" content="">
```
Comment (spolischook) [inline src/Geekhub/ResourceBundle/Resources/views/layout.html.twig]: Add block with default value "Geekhub PHP Developers Team"

File: src/Geekhub/ResourceBundle/Resources/views/layout.html.twig [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
+
+        <title>{% block title %}Welcome{% endblock %}</title>
+
+        <!-- Bootstrap core CSS -->
```
Comment (spolischook) [inline src/Geekhub/ResourceBundle/Resources/views/layout.html.twig]: Remove ALL extra comments

**Pull request merged**